frontend/src/platform/types/track_models/audio_track.ts

```
@@ -0,0 +1,107 @@
+  id: string;
+  name: string;
+  audio_file_storage_key: string;
+  audio_file_format: string;
+  audio_file_size: number;
+  audio_file_duration: number;
+  audio_file_sample_rate: number;
+}
+/**
+ * API request model for deleting an audio track
+ */
+export interface AudioTrackDelete {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)